app/services/update_induction_tutor.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class UpdateInductionTutor < BaseService
+  attr_accessor :school, :email, :full_name
+
+  def initialize(school:, email:, full_name:)
+    @school = school
+    @email = email
+    @full_name = full_name
+  end
+
+  def call
+    ActiveRecord::Base.transaction do
+      @school.induction_tutor.update!(full_name: @full_name,
+                                      email: @email)
+    end
```

Comment:
Doesn't need to be in a transaction for a single update, however i think this needs to check that the person that is being updated does not have participant profiles (ECF or NPQ) and validation data etc and we're changing the name on their user record to a completely different person.